LayoutTests/platform/mac/TestExpectations

```diff
@@ -863,6 +863,40 @@ imported/w3c/web-platform-tests/media-source/mediasource-trackdefaultlist.html [
 media/media-source/media-source-init-segment-duration.html
 # --- End Media Source Tests ---
 
+# --- Start WebVTT Tests ---
+## --- Start W3C Imported WebVTT Tests ---
+
+# imported/w3c/web-platform-tests/webvtt/rendering/cues-with-video tests are marked as [ ImageOnlyFailure ] by default
+# Add newly passing tests to this file as Apple-port-specific bugs are fixed:
```

Comment:
(potentially a separate issue) but inverting this would be better, so new tests that are imported get run

Reply:
There are about 250 tests in this directory - inverting this so that over 200 tests are marked individually as failing could possibly impact readability of this file (but this probably isn't relevant to this patch)